src/components/logged_in_shipping_address/LoggedInShippingAddress.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { RadioField } from '@boldcommerce/stacks-ui';
+import { useCountryInfo, useShippingAddress, useCheckoutStore } from '../../hooks';
+import Address from '../address/Address'
+import './LoggedInShippingAddress.css';
+
+const AddressData = ({ addressInfo }) => {
```

Comment:
Maybe break this out into a separate file and rename this to something else like ReadonlyAddress for example

Reply:
Maybe call this DisplayAddress and use this on the order confirmation page as well as a reusable component. If you do this, you'll likely want to change the classNames on this to be more generic and not tied to CheckoutStep specifically.